dipy/reconst/shm.py

```diff
@@ -52,8 +54,8 @@ def forward_sdeconv_mat(r_rh, n):
         function. Each element `rh[i]` is associated with spherical harmonics
         of degree `2*i`.
```

Comment:
Double backticks or math mode.

Reply:
Hi, the docstrings should be fixed by commit [11beb24](https://github.com/dipy/dipy/pull/2206/commits/11beb2414689604f8c256a4dfb1e703c34a70ff4)!

Reply:
Looks good at first sight. Thanks @CHrlS98 !